client/src/Components/VideoChat/index.tsx

```diff
@@ -83,19 +88,26 @@ export default class VideoChat extends React.Component<VideoChatProps, VideoChat
     }
 
     render() {
+
+        const streams = this.props.streams.concat([this.self!]);
```

Comment:
We should move this line to the componentDidMount function to avoid concat self many times